### PR TITLE
htlwienwest-da:0.1.0

### PR DESCRIPTION
**Pull request by peterw16**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: This is a diploma thesis template for students (most specifically of the HTL Wien West highschool).

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Review by elegaanz — COMMENTED** [review]
I left a few comments, but overall it looks good! I think the name should be different though, so that it has some uniqueness that would make it possible for other students to propose an alternative template without having an unfair naming disadvantage.

File: packages/preview/htlwienwest-da/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+ - `kurzfassung`: `content` - Abstract in german as content block
+ - `abstract`: `content` - Abstract in english as content block
+ - `vorwort`: `content` - The thesis' preface
+ - `danksagung`: `content` - Acknowledgement
+ - `anhang`: `content | none` - Appendix
+ - `literaturverzeichnis`: `function` - The bibliography prefilled with the BibTex file path
+
+The function also accepts a single, positional argument for the body of the paper.
+
+The template will initialize your package with a sample call to the `ieee`function in a show rule. If you want to change an existing project to use thistemplate, you can add a show rule like this at the top of your file:
```
[the remark this answers is not in the document]
Comment (peterw16) [reply packages/preview/htlwienwest-da/0.1.0/README.md:39]: Thank you for the hint!

File: packages/preview/htlwienwest-da/0.1.0/lib/template.typ [hunk 2]
```diff
@@ -0,0 +1,335 @@
+#import "elements.typ" as elems
+#import "settings.typ" as settings
+#import "fontchecker.typ" as fc
+#import "assertions.typ": *
+
+#let diplomarbeit(
+  titel: none, 
+  abteilung: none,
+  unterschrifts_datum: none,
```
[the remark this answers is not in the document]
Comment (peterw16) [reply packages/preview/htlwienwest-da/0.1.0/lib/template.typ]: Changed it to kebab-case style.

**Comment by peterw16** [comment]
@elegaanz Thank you for your review!

> ... make it possible for other students to propose an alternative template without having an unfair naming disadvantage.

Students at HTL WienWest must use one of the official diploma thesis templates. Currently there are templates for Latex and Word. This package will be the third official option using Typst. Therefore, I think a less precise name will make it more difficult for students to find the only accepted template for their thesis.

**Comment by reknih** [comment]
Hey @peterw16! As far as I understood, this template does implement the official style of HTL WienWest dissertations in Typst. Is this implementation itself also endorsed by HTL WienWest (e.g. by a professor or administrative staff)? If so, please send a note to that effect using your academic email address to hello@typst.app and we will approve the name.

If, on the other hand, another member of HTL WienWest could also implement the official dissertation style in a different package and be another viable option for doctoral students, then we maintain that the package name should have a non-descriptive component to allow for a diversity of community implementations. You will still be able to migrate to this name once staff officially endorses this package in particular.

**Pull request merged**